04-One-Dimensional-Kalman-Filters.ipynb

```diff
@@ -1290,7 +1290,7 @@
     }
    ],
    "source": [
-    "book_plots.plot_errorbars([(160, 8, 'A'), (170, 8, 'B')], xlims=(150, 180))"
+    "book_plots.plot_errorbars([(160, 3, 'A'), (170, 9, 'B')], xlims=(150, 180))"
```

Comment:
If you look at the text below it. (Search for "157 to 163")
This will be the correct error value and it's consistent with the next code cell.